Sources/Features/Client/Model/AttachmentBookmarkHandler.swift

```diff
@@ -10,8 +10,9 @@ enum AttachmentBookmarkHandlerImpl: AttachmentBookmarkHandler {
         var attachmentsBookmarksDict = Bookmarks()
         for attachment in attachments {
             do {
-                let bookmark = try attachment.value.bookmarkData(options: URL.BookmarkCreationOptions.minimalBookmark)
-                attachmentsBookmarksDict[attachment.key] = bookmark
+                let bookmark = try attachment.bookmarkData(options: URL.BookmarkCreationOptions.minimalBookmark)
```

Comment:
Short-hand syntax should work as well:

```suggestion
                let bookmark = try attachment.bookmarkData(options: .minimalBookmark)
```

Reply:
Will do, thank you!